src/app/blog/blog.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { Title, Meta } from '@angular/platform-browser';
+import { ActivatedRoute, Router } from '@angular/router';
+import { GoogleAnalyticsService } from '../services/analytics/googleAnalyticsService';
+import { BlogRepository, IBlogDetail } from '../services/blogs/blogRepository';
+import { GlobalConstantsRepository } from '../services/shared/globalConstantsRepository'
+import { WorkshopsListComponent } from '../workshops-list/workshops-list.component';
+
+@Component({
+    templateUrl: './blog.component.html',
+    styleUrls: ['./blog.component.scss']
+})
+
+export class BlogComponent {
+
+    blogId: string;
+    // @ViewChild(WorkshopsListComponent) workshopsListChildComp:WorkshopsListComponent;
+
+    blog: IBlogDetail;
+
+    constructor(private blogRepository: BlogRepository,
+        private globalConstantsRepository: GlobalConstantsRepository,
+        private route: ActivatedRoute,
+        gaService: GoogleAnalyticsService,
+        title: Title,
+        meta: Meta) {
+        gaService.trackPageView('blogs');
+        title.setTitle('Photography blogs - Pixelated Planet')
```

Comment:
change to title of blog. Also change other meta tag properties to correct values